src/components/Sorting/sorting.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import './sorting.scss';
+
+export const Sorting = () => (
+  <div className="sorting">
+    <span>sort by</span>
+    <select>
+      <option value="value1" defaultValue>release date</option>
```

Comment:
how about using `array.map` here in the future? The same as with movie cards and filters.

Reply:
fixed